SavedInstances/Modules/WorldBoss.lua

```diff
@@ -70,4 +70,6 @@ SI.WorldBosses = {
   [9002] = { quest=47461, name=L["Lord Kazzak"],         expansion=8, level=60, remove=true }, -- Lord Kazzak
   [9003] = { quest=47462, name=L["Azuregos"],            expansion=8, level=60, remove=true }, -- Azuregos
   [9004] = { quest=47463, name=L["Dragon of Nightmare"], expansion=8, level=60, remove=true }, -- Dragon of Nightmare
+  -- Eastern Plaguelands (Shadowlands pre-patch event)
+  [2381] = { quest=60542, name=L["Nathanos Blightcaller"], expansion=7, level=50 }, -- Nathanos Blightcaller

```

Comment:
index here should use 9005 as dummy index for Nathanos isn't in Encounter Journal and don't have encounter index

Reply:
Oops, fixed!